haiku-apps/copynametoclipboard/copynametoclipboard-1.0.1.recipe

```diff
@@ -9,7 +9,10 @@ COPYRIGHT="2000 Werner Freytag
 	2012 Humdinger"
 LICENSE="MIT"
 REVISION="2"
-SOURCE_URI="$(HOMEPAGE)/archive/1da1d52429db53f15c3debf817e982d9ce6346b4.tar.gz"
+srcGitRev="1da1d52429db53f15c3debf817e982d9ce6346b4"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="1da1d52429db53f15c3debf817e982d9ce6346b4"
```

Comment:
IIRC it only calculates the checksum if there is a CHECKSUM in the recipe, so putting a bad one should make it do so.
Also the git revision is a checksum of the commit itself, the archive has nothing to do with it as it's created afterward.